automation/WinFormsAutomationApp/WinFormsAutomationApp.csproj

```diff
@@ -97,7 +91,6 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>Resources.resx</DependentUpon>
     </Compile>
-    <None Include="272MSSharedLibSN2048.snk" />
```

Comment:
It's not necessary to include the snk file in the project - setting the property is enough (although if we did want to see it we'd add it to the common settings file).